ArrCharOps.java

```diff
@@ -126,7 +180,22 @@ public static long hashCode(char[] arr) {
      *         return -2 if there is an error with the input.
      */
     public static int compareTo(String str1, String str2) {
-        // Replace the following statement with your code
-        return 0;
-    }
+    if(str1 == "" || str2 == "") return -2;
```

Comment:
`str1.length() == 0` or `str1.equals("")`
Either way it is ok. 
`str1 == ""` is not.